Stepippo/Classes/Views/CheckButton.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ProgCheckButton.swift
+//  Stepippo
+//
+//  Created by 加藤彩那 on 5/9/31 H.
+//  Copyright © 31 Heisei Yasasii-kai. All rights reserved.
+//
+
+import UIKit
+
+class CheckButton: UIButton {
+    // Images
+    let checkedImage = UIImage(named: "Checkbox")! as UIImage
+    let uncheckedImage = UIImage(named: "UnCheckbox")! as UIImage
+        
+    // Bool property
+    var isChecked: Bool = false {
+        didSet{
+            if isChecked == true {
```

Comment:
条件に使うBoolは以下の書き方でOKです👌
```suggestion
            if isChecked {
```